.github/workflows/dev_build.yaml

```diff
@@ -6,18 +6,11 @@ on:
       - test/*
```

Comment:
does this run when we want? maybe?

```
on:
  push:
    paths:
      - 'tests/'
```

Reply:
maybe since this doesn't need to run a lot, it could be:

```
on:
  push:
    paths:
      - pyproject.toml
      - Makefile
```

Reply:
It was so i could run the actions on my personal repo before opening the pull request, was planning on switching before opening the PR but forgot, thanks for the reminder.

Reply:
Hmm, maybe just `pyproject` or `package.json`